dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -135,7 +142,41 @@ private static void removeAnyOldMetadata() throws DotDataException {
                String.format(" `%d` old metadata entries removed and `%d` old content files from the starter." , tuple._1 , + tuple._2)
             );
         }catch (ExecutionException | InterruptedException e){
-            Logger.error(DotCMSInitDb.class,"");
+            Logger.error(DotCMSInitDb.class,"An error occurred removing old metadata ",e);
```

Comment:
did you check if this message becomes so noisy? You changed it to print the stack trace also

Reply:
it's not noisy at all